docs/guides/runtime-options-overview.mdx

```diff
@@ -18,288 +19,298 @@ service = QiskitRuntimeService(channel="ibm_cloud", channel_strategy="q-ctrl")
 
-### Options class
+1. Any options set after primitive initialization.
+2. Any options set during primitive initialization.
 
-When creating an instance of the `Estimator` or `Sampler` class, you can pass in the `options` you created in the options class. Those options will then be applied when you use `run()` to perform the calculation.  Example:
+<Admonition type="caution" title="Important">
+Shots (Sampler) or precision (Estimator) set in `run()` are not considered options, but take precidence over values set for `default_shots`or `default_precision`. See the [Run() method](specify-runtime-options#run-method) section for full details.
+</Admonition>
 
-<Tabs>
-  <TabItem value="PrimV2" label="V2 primitives">
-    `SamplerV2` and `EstimatorV2` have separate options classes that do not need to be instantiated.  You can see the available options and update option values during or after primitive initialization.  Those options will then be applied when you use `run()` to perform the calculation.  Example:
+In the following example, these values would be used for the `estimator.run` job: `shots = 128` and `measure_mitigation = True`.
 
-```python
-estimator = Estimator(mode=backend)
+``` python
+# Setting options during primitive initialization
+estimator = Estimator(backend, options={'resilience_level': 2,'resilience': {'measure_mitigation': False}})
 
 # Setting options after primitive initialization
 # This uses auto complete.
+estimator.options.resilience.measure_mitigation = True
 estimator.options.default_shots = 4000
-```
-  </TabItem>
-
-  <TabItem value="PrimV1" label="V1 primitives">
-When creating an instance of the `Estimator` or `Sampler` class, you can pass in the `options` you created in the options class. Those options will then be applied when you use `run()` to perform the calculation.  Example:
 
-```python
-estimator = Estimator(backend=backend, options=options)
-result = estimator.run(circuit, observable).result()
-print(f">>> Metadata: {result.metadata[0]}")
+# Sample two circuits at 128 shots each.
+estimator.run([(circuit1, observable1), (circuit2, observable2)], shots=128)
 ```
-  </TabItem>
-</Tabs>
 
-### Primitive initialization
+#### Special cases
 
-You can specify options when initializing the primitive.
+The `default_shots` and `resilience_level` options require special consideration:
 
+If you set `default_shots` in an estimator, it overrides any value set for `default_precision`.
 
-<Tabs>
-  <TabItem value="PrimV2" label="V2 primitives">
- ```python
-from qiskit_ibm_runtime import QiskitRuntimeService
-from qiskit_ibm_runtime import EstimatorV2 as Estimator
+The `resilience_level` is used to set a base configuration on any relevant options, and then any other user-specified options override that configuration. So it would be possible to undo the effect of setting the resilience level by reversing all of its settings manually.
```

Comment:
```suggestion
The `resilience_level` is used to set a base configuration on any relevant options. For example, setting the resilience level to 2 in the estimator tells the runtime estimator to start with a base options configuration that enables TREX, gate twirling, and ZNE with sensible defaults. Any other user-specified options override that configuration. So it would be possible to undo the effect of setting the resilience level by reversing all of its settings manually.
```